make/cluster.sh

```diff
@@ -163,7 +163,12 @@ setup_kind() {
 
   # (1) Does the kind cluster already exist?
   if ! kind get clusters -q | grep -q "^$kind_cluster_name\$"; then
-    trace kind create cluster --config make/config/kind/v1beta2.yaml \
+    # Kind with Kubernetes v1.18 and v1.19 need different config files
+    config="config_etcd_no_fsync"
+    if [[ "$k8s_version" == 1.18 || "$k8s_version" == 1.19 ]]; then
```

Comment:
cert-manager 1.8 no longer supports Kubernetes 1.18 and our e2e test no longer run against 1.18
however 1.18 is still supported by these scripts

I think the remaining references to 1.18 and the support for it could be removed all together https://github.com/cert-manager/cert-manager/issues/5056